src/renderers/WebGLRenderer.js

```diff
@@ -304,6 +304,7 @@ function WebGLRenderer( parameters ) {
 		_this.renderLists = renderLists;
 		_this.state = state;
 		_this.info = info;
+		_this.objects = objects;
```

Comment:
Oh wait. Objects was not public before?

Reply:
Nope. Um...yeah making `objects` public is not an ideal solution. But ideally `WebGLShadows` can use the interface of `WebGLRenderer` to access the latest `WebGLObjects` instance?

Reply:
Or we just do something like:
```js
_this.__objects = objects;
```

Reply:
Regarding nomenclature, I would name it `webglObjects`.

Reply:
> _this.renderLists = renderLists;
> _this.state = state;
> _this.info = info;

These properties do not user the `webgl` prefix, too. So I think `objects` is more consistent to the current nomenclature of `WebGLRenderer`.